h2o-r/h2o-package/R/models.R

```diff
@@ -3096,7 +3101,8 @@ h2o.cross_validation_predictions <- function(object) {
 #' }
 #' @export
 
-h2o.partialPlot <- function(object, data, cols, destination_key, nbins=20, plot = TRUE, plot_stddev = TRUE, weight_column=-1, include_na=FALSE, user_splits=NULL) {
+h2o.partialPlot <- function(object, data, cols, destination_key, nbins=20, plot = TRUE, plot_stddev = TRUE,
```

Comment:
it seems like the new parameter is called save_to in R and save_to_file in Python, is that intended?

Reply:
Yes, this is intentional, as there is rarely one file in R.

Reply:
Makes sense, thank you @Pscheidl

Reply:
Do we have any existing naming conventions in R that we should consider?  (regarding parameters that save to disk...)